hypha/templates/base.html

```diff
@@ -125,6 +122,11 @@
         {% include "includes/_modal-placeholder.html" %}
         {% include "includes/_toast-placeholder.html" %}
 
+        <!-- We still need jQuery for select2, daterangepicker and a few custom scripts. -->
+        <!-- Please do not add new features that require jQuery! -->
+        <script src="{% static 'js/vendor/jquery.min.js' %}"></script>
+
+        {{ form.media.js }}
```

Comment:
@theskumar Is this ok, any drawbacks?

Seems to have no ill effect. On many pages there are no "form.media*" but then it just outputs nothing silently.

This is needed on many htmx loaded dialogs etc. Loading it here we make sure it is loaded after jQuery and avialevel when needed.